adapters/sovrn/sovrn_test.go

```diff
@@ -0,0 +1,279 @@
+package sovrn
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/pbs"
+
+	"context"
+	"net/http"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/cache/dummycache"
+	"github.com/prebid/prebid-server/config"
+)
+
+func TestJsonSamples(t *testing.T) {
+	sovrnAdapter := NewSovrnBidder(new(http.Client), "http://sovrn.com/test/endpoint")
+	adapterstest.RunJSONBidderTest(t, "sovrntest", sovrnAdapter)
+}
+
+// ----------------------------------------------------------------------------
+// Code below this line tests the legacy, non-openrtb code flow. It can be deleted after we
+// clean up the existing code and make everything openrtb.
+
+var testSovrnUserId = "SovrnUser123"
+var testUserAgent = "user-agent-test"
+var testUrl = "http://news.pub/topnews"
+var testIp = "123.123.123.123"
+
+func TestSovrnAdapterNames(t *testing.T) {
+	adapter := NewSovrnAdapter(adapters.DefaultHTTPAdapterConfig, "http://sovrn/rtb/bid")
+	adapters.VerifyStringValue(adapter.Name(), "sovrn", t)
+	adapters.VerifyStringValue(adapter.FamilyName(), "sovrn", t)
+}
+
+func TestSovrnAdapter_SkipNoCookies(t *testing.T) {
+	adapter := NewSovrnAdapter(adapters.DefaultHTTPAdapterConfig, "http://sovrn/rtb/bid")
+	adapters.VerifyBoolValue(adapter.SkipNoCookies(), false, t)
+}
+
+func TestSovrnOpenRtbRequest(t *testing.T) {
+	service := CreateSovrnService(adapters.BidOnTags(""))
+	server := service.Server
+	ctx := context.TODO()
```

Comment:
I think `context.Background()` is more right here. I know other bidders use `context.TODO()`, but... looking at the docs, I don't think it's appropriate: https://golang.org/pkg/context/#TODO

If we did run some static analysis to look for issues, I don't think we'd want it to trigger here..